src/app/playground-components.ts

```diff
@@ -1242,6 +1248,12 @@ export const PLAYGROUND_COMPONENTS: ComponentLink[] = [
         component: 'TagRemovableComponent',
         name: 'Tag Removable',
       },
+      {
+        path: 'tab-github-test.component',
+        link: '/tag/tab-github-test.component',
+        component: 'TagGithubTestComponent',
+        name: 'Tag Github Test',
+      },
```

Comment:
Unrelated change. Rerun `npm run gen:playground` to regenerate this list without unnecessary entries and commit changes.